opencti-platform/opencti-graphql/src/modules/playbook/playbook-components.ts

```diff
@@ -580,26 +588,42 @@ const PLAYBOOK_UPDATE_KNOWLEDGE_COMPONENT: PlaybookComponent<UpdateConfiguration
           .map((action) => {
             const attrPath = computeAttributePath(type, action.attribute);
             const multiple = isAttributeMultiple(type, action.attribute);
-            const numeric = isAttributeNumeric(type, action.attribute);
-            return ({ action, multiple, numeric, attrPath, path: `/objects/${index}${attrPath}` });
+            const attributeType = getAttributeType(type, action.attribute);
+            return ({ action, multiple, attributeType, attrPath, path: `/objects/${index}${attrPath}` });
           })
           // Unrecognized attributes must be filtered
           .filter(({ attrPath, multiple }) => isNotEmptyField(multiple) && isNotEmptyField(attrPath))
           // Map actions to data patches
-          .map(({ action, path, multiple, numeric }) => ({
-            op: action.op,
-            path,
-            // eslint-disable-next-line no-nested-ternary,max-len
-            value: multiple ? action.value.map((o) => (numeric ? Number(o.patch_value) : o.patch_value)) : numeric ? Number(R.head(action.value)?.patch_value) : R.head(action.value)?.patch_value
-          }));
+          .map(({ action, path, multiple, attributeType }) => {
+            if (multiple) {
+              const currentValues = jsonpatch.getValueByPointer(bundle, path) ?? [];
+              const actionValues = action.value.map((o) => {
+                // If value is an id, must be converted to standard_id has we work on stix bundle
+                if (cacheIds.has(o.patch_value)) return (cacheIds.get(o.patch_value) as BasicStoreCommon).standard_id;
```

Comment:
what is cacheIds ? I don't understand this part.

Reply:
Related to // If value is an id, must be converted to standard_id has we work on stix bundle
We have filter like markings using internal_id for configuration. They need to be translated to standard to allow correct filtering